src/main/java/tds/content/controllers/ContentController.java

```diff
@@ -51,11 +55,11 @@ public ContentController(final ContentService contentService) {
     @ResponseBody
     public ResponseEntity<ITSDocument> getItemDocument(@RequestParam final String itemPath, @RequestParam(required = false) final String contextPath,
                                                        @RequestParam(required = false) final boolean oggAudioSupport,
-                                                       @RequestBody final AccLookup accLookup) {
+                                                       @RequestBody final AccLookup accLookup) throws IOException {
```

Comment:
Or run a custom one that handles all these one off exceptions rather than try/catching them all.